UPGRADING.md

```
@@ -85,6 +85,22 @@ Additionally, a simplification of the start command leads to the following chang
+```diff
+import (
++ 	txdecode "cosmossdk.io/x/tx/decode"
+)
++	txDecoder, err := txdecode.NewDecoder(txdecode.Options{
++		SigningContext: signingCtx,
++		ProtoCodec:     appCodec,
++	})
++	if err != nil {
++		panic(err)
++	}
+```
```

Markdownlint

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)